sspl_test/conf/sspl_tests.conf.yaml

```diff
@@ -0,0 +1,39 @@
+SYSTEM_INFORMATION:
+   rack_id: 'RC01'
+   site_id: 'DC01'
+   cluster_id: 'CC01'
+   node_id: 'SN01'
+SSPL-TESTS_SETTING:
+   modules:
+      - RabbitMQingressProcessorTests
+      - RabbitMQegressProcessor
+RABBITMQEGRESSPROCESSOR:
+   virtual_host: SSPL
+   queue_name: actuator-req-queue
+   exchange_name: sspl-in
+   routing_key: actuator-req-key
+   username: sspluser
+   password: gAAAAABgB8tRJLZBfI6uCAtt4B8YYjveDvhHnFCbeKYsXuYxSLl3KZE5Tp94Jl2qQq9-X7x7_CinZZSh4cm9rMJ4G3HKz36I-Q==
+   message_signature_username: sspl-ll
+   message_signature_token: ALOIUD986798df69a8koDISLKJ282983
+   message_signature_expires: '3600'
+   primary_rabbitmq_host: localhost
+RABBITMQINGRESSPROCESSORTESTS:
+   virtual_host: SSPL
+   queue_name: sensor-queue
+   exchange_name: sspl-out
+   routing_key: sensor-key
+   username: sspluser
+   password: gAAAAABgB8tRJLZBfI6uCAtt4B8YYjveDvhHnFCbeKYsXuYxSLl3KZE5Tp94Jl2qQq9-X7x7_CinZZSh4cm9rMJ4G3HKz36I-Q==
```

Comment:
Do we need this password?

Reply:
It is being used in rabbitmq test processors